CONTRIBUTING.md

```diff
@@ -18,4 +18,54 @@ After you have contributed something check your code by using:
 $ npm run linting | npm test
 ```
 
+## Debugging
+
+Included in the modules is some basic debugging logs. You can set each of these individually to target the log of interest:
+
+```js
+var diacritics = require("diacritics-transliterator");
+diacritics.debug = {
+    server: true, // show server & cache interactions
+    regexp: true, // show resulting regular expressions
+    regexpTests: true, // show all regular expression tests
+    placeholder: true // show string breakdown and results
+};
+diacritics.getVariant("de");
+// logged as: "http://api.diacritics.io/v1/?language=de; response: loaded"
+diacritics.getVariant("de");
+// logged as: "http://api.diacritics.io/v1/?language=de; {loaded from cache}"
+```
+
```

Comment:
I guess that this code is from a users point of view as we don't require the module in `index.js`. If so, then the debug option is missing in the README.md